### PR TITLE
Adds helianthus, a hallucinogenic absinthe cocktail.

### PR DESCRIPTION
**Pull request by Sylphet**
## About The Pull Request

This adds a new, hallucinogenic cocktail, helianthus, made from one part absinthe, one part sugar, and one part mindbreaker.

## Why It's Good For The Game

A lot of our existing drinks don't really do much. This one causes hallucinations for the drinker, as well as adds another use for absinthe, used in only two other drinks. It's original, and is nice flavour.

image

## Changelog

:cl:
add: adds helianthus cocktail
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
Wtf adding fulp admins to the game

**Comment by Lyn505** [comment]
https://tenor.com/view/based-kneel-dark-souls-gif-24182729

**Comment by Sheits** [comment]
REAL

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2812,3 +2812,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	taste_description = "golden memories"
+	glass_icon_state = "helianthus"
+	glass_name = "Helianthus"
+	glass_desc = "Another reason to cut off an ear..."
+	var/hal_amt = 4
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/helianthus/on_mob_life(mob/living/carbon/drinker, delta_time, times_fired)
+	if(DT_PROB(5, delta_time))
+		drinker.hallucination += hal_amt
```
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2831]: you need to do a log algorithm or a cap to ensure you can't stack hal_amt to ludicrous levels by drinking or forcefeeding others

**Comment by Farquaar** [comment]
I'm not really a fan of the sprite. Why is the drink orange/yellow?

If it's going to be an absinthe-based "green fairy" type of drink, shouldn't it be, y'know, green?

**Comment by Sylphet** [comment]
> I'm not really a fan of the sprite. Why is the drink orange/yellow?
> 
> If it's going to be an absinthe-based "green fairy" type of drink, shouldn't it be, y'know, green?
> 
> Actually, scratch that. This gives me an idea.

I'm not _super_ happy with the sprite myself, but it's yellow and orange because it's themed after a sunflower. Absinthe has a reputation as an artist's drink, so... Van Gogh, sunflowers, absinthe... you get the point.

**Comment by EricZilla** [comment]
Does this still count as mindbreaker for suppressing reality disassociation? Of so that's a fun way to take your medicine.

**Comment by Sylphet** [comment]
> Does this still count as mindbreaker for suppressing reality disassociation? Of so that's a fun way to take your medicine.

It does not but only because I'm too stupid to code that in. I'd be very happy for someone to make a PR for that !

**Comment by GreatKingWombat** [comment]
> I'm not really a fan of the sprite. Why is the drink orange/yellow?
> 
> If it's going to be an absinthe-based "green fairy" type of drink, shouldn't it be, y'know, green?
> 
> Actually, scratch that. This gives me an idea.

Actually, it would be a yellowish white. 

Source: I actually drink absinthe.

**Pull request merged**